test/sub_examples/fixtures/rapid_scada_post_recv.html

```diff
@@ -0,0 +1,181 @@
+<!doctype html>
```

Comment:
Do you really need this file ? I could not see where you open it (just had a quick look at the code).

Reply:
It's used in testing, not in the actual example.  It enables:
- Testing not hitting https://rapidscada.net/modbus/ every `pytest` invocation
- Devs can see what the payload looks like

Reply:
Removed this file entirely (along with tests) per [this request](https://github.com/pymodbus-dev/pymodbus/pull/1665#discussion_r1267615225)